bin/test-wordpress-coding-standards.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
@stevector I was having issues with the build failing if code sniffing threw errors - which is why I moved it to a script to check the output.